cli/sync_service/object.go

```diff
@@ -130,15 +130,19 @@ func ObjectList(org string, userPw string, objType string, objId string, destPol
 
 		mmsObjects := make([]MMSObjectInfo, 0)
 		c := make(chan MMSObjectInfo)
-		input_chan := make(chan common.MetaData)
 
 		for i := 0; i < len(batches); i++ {
-			for _, _ = range batches[i] {
+			for _, obj := range batches[i] {
+				obj := obj
```

Comment:
What is this doing? Is it necessary?

Reply:
It is making a copy (it's called a shadow variable) so that the copy can be used in the go routine. For loop variables are reused on each iteration, so that would cause a problem if the go routine did not use it's own copy.